src/promptflow/promptflow/core/__init__.py

```diff
@@ -7,9 +7,9 @@
 
 # flake8: noqa
 from promptflow._core.tool import ToolProvider, tool
-from promptflow.core._flow import AsyncFlow, Flow
+from promptflow.core._flow import AsyncFlow, Flow, Prompty
 
 # backward compatibility
 log_flow_metric = log_metric
 
-__all__ = ["log_metric", "ToolProvider", "tool", "Flow", "AsyncFlow"]
+__all__ = ["log_metric", "ToolProvider", "tool", "Flow", "AsyncFlow", "Prompty"]
```

Comment:
Let's expose this when the feature fully developed

Reply:
remove it